tests/Feature/LangTest.php

```diff
@@ -35,5 +38,21 @@ public function testLang(): void
 				static::assertArrayHasKey($key, $locale, 'Language ' . $lang_test->code() . ' is incomplete.');
 			}
 		}
+
+		static::assertEquals('en', Lang::get_code());
+		static::assertEquals('OK', Lang::get('SUCCESS'));
+
+		Configs::where('key', '=', 'lang')->delete();
+		$lang = new \App\Locale\Lang(new LangFactory());
+		self::assertEquals('en', $lang->get_code());
+
+		DB::table('configs')->insert([
+			[
+				'key' => 'lang',
+				'value' => 'en',
+				'confidentiality' => 0,
+				'cat' => 'Gallery',
+			],
+		]);
```

Comment:
Only after inspecting the code I got what you wanted to test here. Instead of adding this test code to the already existing but imprecisely named method `testLang` I would rather move this code into its own test method like `testEnglishAsFallbackIfLangConfigIsMissing`. I know this method name is rather long, but it really helps reader to understand what you want to test. `testLang` is simply so untelling.

Moreover, there are authors who advocate that there must be only a single assertion per test as otherwise the test is too large for being a single test. I wouldn't go that far, because sometime you simply need multiple assertions for a test to make sense. Hence, I only consider this only a rule of thumb, but the underlying idea is correct.

Lychee is a prime example which simply puts to many and unrelated tests into a single test method. While it is tedious to try to repair that for the existing tests, we should try to do better for newly written tests.